app/components/chat-message/chat-message.tsx

```diff
@@ -136,25 +133,29 @@ export const ChatMessage: React.FC<Props> = ({ sender, seckey, recipient, messag
 
   useEffect(() => {
     isMounted.current = true
-    if (message.text && seckey?.key && recipient?.hexpubkey() && sender?.hexpubkey()) {
+    if (
+      message.text &&
+      senderSecretKey &&
+      nip19.decode(senderSecretKey).data.toString()
+    ) {
       const senderSec: secureKeyPair = {
-        seckey: seckey?.key,
-        pubkey: sender?.hexpubkey(),
+        seckey: nip19.decode(senderSecretKey).data.toString(),
+        pubkey: nip19.decode(senderPubKey).data.toString(),
       }
       const recipientSec: secureKeyPair = {
         seckey: "39c34c3f2600a36d582cf9fca1bfffc102f0532d4c1ba74a2d1aa5afcb061c31",
         pubkey: "7ebc48d3e51a4d81719b7c1d601fd6f55b08e83d731e2eda27aacc543cbc1840",
```

Comment:
Do we still need this?

Reply:
no, but i've kept it will update it in my chat window re-work PR